testing/asserts.ts

```diff
@@ -97,7 +97,9 @@ function buildMessage(
         ? createColor(detail.type, { background: true })(detail.value)
         : detail.value
     ).join("") ?? result.value;
-    diffMessages.push(c(`${createSign(result.type)}${line}`));
+    diffMessages.push(c(`${
+      createSign(result.type)
+    }${line}`));
```

Comment:
Unrelated change?

Reply:
I've seen this change recently in some other PR, seems to be related to dprint. CC @dsherret